rosys/vision/calibration.py

```diff
@@ -242,10 +243,11 @@ def project_from_image(self, image_coordinates: Point | np.ndarray, target_heigh
                 return None
             return Point3d(x=world_points[0, 0], y=world_points[0, 1], z=world_points[0, 2])  # pylint: disable=unsubscriptable-object
 
+        world_extrinsics = self.extrinsics.resolve()
         image_rays = self.points_to_rays(image_coordinates.astype(np.float32).reshape(-1, 1, 2))
-        objPoints = image_rays @ self.rotation_array.T
-        Z = self.extrinsics.translation[-1]
-        t = np.array(self.extrinsics.translation)
+        objPoints = image_rays @ world_extrinsics.rotation.matrix.T
+        Z = self.extrinsics.z
```

Comment:
```suggestion
        Z = world_extrinsics.z
```

@NiklasNeugebauer Shouldn't we use `world_extrinsics` here as well?